### PR TITLE
refactor(locksmith): Upgrade express

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

Upgrade express

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "dayjs": "1.11.6",
     "dotenv": "16.0.3",
     "ethers": "5.7.2",
-    "express": "4.18.2",
+    "express": "5.0.0-beta.1",
```
Comment (julien51) [inline locksmith/package.json:59]: beta? risky?

**Review by julien51 — APPROVED** [review]
LGTM

File: locksmith/package.json (hunk 1 above)
Comment (searchableguy) [reply locksmith/package.json:59]: it's been in beta since 4-5 years

**Comment by julien51** [comment]
Screen Shot 2022-12-08 at 6 14 41 PM

I'll merge because this guy is stuck.

**Pull request merged**